pkg/storaged/utils.js

```diff
@@ -862,13 +887,28 @@
             });
         }
 
-        if (fsys && fsys.MountPoints.length > 0) {
-            fsys.MountPoints.forEach(mp => {
-                const mpd = decode_filename(mp);
-                const children = find_children_for_mount_point(client, mpd, null);
+        // HACK: get_active_usage is used for mounting and formatting so we use the absence of the subvol argument
+        // to figure out that we want to format this device.
+        // This is separate from the if's below as we also always have to umount the filesystem.
+        if (btrfs_volume && !subvol) {
+            usage.push({
+                level,
+                usage: 'btrfs-device',
+                block,
+                btrfs_volume,
+                location: btrfs_volume.data.label || btrfs_volume.data.uuid,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19690-20240111-120527-6a639276-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)